src/FixerConfiguration/FixerOptionBuilder.php

```diff
@@ -108,6 +113,18 @@ public function setNormalizer(\Closure $normalizer)
         return $this;
     }
 
+    /**
+     * @param string $preferredName
+     *
+     * @return $this
+     */
+    public function setAlias($preferredName)
```

Comment:
this is not internal class, in perfect world, we don't want to have aliases. For that ,we don't want to expose this "new feature" to users and allow them to use it - as this means we need to support it in long term.
Instead, make new, internall class perhaps?
take a look at `FixerConfigurationResolverRootless` - when we wanted all options to have root, but not all of them did, we added temporary class to ease development of it.

Reply:
Yeah I thought as much! Thanks for the pointer :)

Reply:
I've implemented decorator classes for `FixerOption` and `FixerOptionBuilder`, I think it solves the issues you brought up!